### DISCUSSION TITLE
Can I reduce the size of a PDF made with 600 images without reduce the quality too much?

### DISCUSSION THREAD

**Discussion by trnkobayashi** [ImageMagick/ImageMagick#6904] [Help] [answered]
### ImageMagick version

7.1.1-21

### Operating system, version and so on

Windows 10

### Description

I tried something like:
magick convert input.pdf -resize 50% "output.pdf"
My PDF goes from 170 MB to 60 MB, it's good but the quality is worse btw

**Comment by fmw42** [comment, marked as answer]
For IM 7, use magick, not magick convert.

Try

`magick -density 288 input.pdf -resize 12.5% output.pdf`

This is a supersampling.  First use density 72 (default) * 4 = 288 to increase size. Then resize by 1/4 * 1/2 = 12.5%.  If the image is too small, then just increase the resize percent.

**Reply by trnkobayashi** [reply]
I run that command and it shows the error:

>magick.exe: unable to extend cache 'C:/Users/Trn_Huy/AppData/Local/Temp/magick-u0jzfhW6kxbskY9r5tEwBZbrRrpehYUG295': No space left on device @ error/cache.c/OpenPixelCache/3925.

Does it mean my PC does not have enough RAM?

**Reply by fmw42** [reply]
No. It is a disk issue with your Temp directory.  Go to C:/Users/Trn_Huy/AppData/Local/Temp/ and clear out all the Imagemagick files (they start with magick-XXXXXX)
